docs/tutorial/testing.rst

```diff
@@ -9,45 +9,53 @@ modifications in the future.  The application above is used as a basic
 example of how to perform unit testing in the :ref:`testing` section of the
 documentation.  Go there to see how easy it is to test Flask applications.
 
-Adding Tests to flaskr
-======================
+Adding tests to flaskr
+----------------------
 
-Assuming you have seen the testing section above and have either written
+Assuming you have seen the :ref:`testing` section and have either written
 your own tests for ``flaskr`` or have followed along with the examples
 provided, you might be wondering about ways to organize the project.
 
-One possible and recommended project structure is::
```

Comment:
Which part exactly? "and recommended"? I did not want to say something that wasn't necessarily true. But if it is a recommended structure, sure I will put that back in. Also I think the "----" makes more sense then "====", for lack of better terms using a subsection as opposed to another section.